cli/local/model_cache.go

```diff
@@ -18,146 +18,123 @@ package local
 
 import (
 	"fmt"
-	"os"
 	"path/filepath"
 	"strings"
-	"time"
 
-	"github.com/cortexlabs/cortex/pkg/lib/archive"
-	"github.com/cortexlabs/cortex/pkg/lib/aws"
-	"github.com/cortexlabs/cortex/pkg/lib/cron"
+	"github.com/cortexlabs/cortex/pkg/consts"
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 	"github.com/cortexlabs/cortex/pkg/lib/files"
-	"github.com/cortexlabs/cortex/pkg/lib/print"
 	"github.com/cortexlabs/cortex/pkg/lib/sets/strset"
 	s "github.com/cortexlabs/cortex/pkg/lib/strings"
 	"github.com/cortexlabs/cortex/pkg/types/spec"
 	"github.com/cortexlabs/cortex/pkg/types/userconfig"
 )
 
-func CacheModels(apiSpec *spec.API, awsClient *aws.Client) ([]*spec.LocalModelCache, error) {
-	modelPaths := make([]string, len(apiSpec.Predictor.Models))
-	for i, modelResource := range apiSpec.Predictor.Models {
-		modelPaths[i] = modelResource.ModelPath
-	}
+func CacheLocalModels(apiSpec *spec.API, onlyLocalModels bool) error {
```

Comment:
With this PR, `onlyLocalModels` would never be set to false. I remember needing this at some point. I think it may be an artifact from when the S3 models were still cached (as part of this PR).